stackslib/src/chainstate/burn/operations/leader_block_commit.rs

```diff
@@ -774,6 +775,7 @@ impl LeaderBlockCommitOp {
     ) -> Result<SortitionId, op_error> {
         let tx_tip = tx.context.chain_tip.clone();
         let intended_sortition = match epoch_id {
+            StacksEpochId::Epoch30 => todo!(), // Need input here
```

Comment:
This should have the same behavior as Epochs 2.1-2.4.

Reply:
Okay, added 3.0 to their match arm.